world/world_countries.js

```diff
@@ -0,0 +1,230 @@
+/*!
+ *
+ * Jquery Mapael - Dynamic maps jQuery plugin (based on raphael.js)
+ * Requires jQuery and Mapael >=2.0.0
+ *
+ * Map of World countries with Antarctica
+ *
+ * Equirectangular projection
+ * Uses the ISO 3166-1 alpha-2 for country names (https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
+ *
+ * @author Vincent Brouté
+ * @source https://commons.wikimedia.org/wiki/File:BlankMap-World6-Equirectangular.svg
+ */
+(function (factory) {
+    if (typeof exports === 'object') {
+        // CommonJS
+        module.exports = factory(require('jquery'), require('jquery-mapael'));
+    } else if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define(['jquery', 'mapael'], factory);
+    } else {
+        // Browser globals
+        factory(jQuery, jQuery.mapael);
+    }
+}(function ($, Mapael) {
+
+    "use strict";
+    
+    $.extend(true, Mapael,
+        {
+            maps :  {
+                world_countries : {
+                    width : 999.29852,
+                    height : 392.03476,
+                    getCoords : function (lat, lon) {
```

Comment:
Should we still use comma-first style?

Reply:
You are right, it comes from the getCoords.php tool on mapael-documentation repository, I will make a PR to fix it.